src/widget.ts

```diff
@@ -20,9 +21,17 @@ export class BlocklyEditor extends DocumentWidget<BlocklyPanel, DocumentModel> {
 
     // Create and add a button to the toolbar to execute
     // the code.
-    // Example: https://github.com/jupyterlab/extension-examples/blob/9c35013ce5da125f1b5865b3f7cbb301970d5970/toolbar-button/src/index.ts#L44-L52
-    // (this.content.layout as BlocklyLayout).run();
-    // this.toolbar.addItem('run', button);
+    const runCode = () => {
+      (this.content.layout as BlocklyLayout).run();
+    };
+    const button = new ToolbarButton({
+      className: 'run-output-button',
+      label: 'Run Code',
+      onClick: runCode,
+      tooltip: 'Run Code',
+    });
+    button.node.style.backgroundColor = 'var(--md-green-500)';
+    this.toolbar.addItem('run', button);
```

Comment:
This looks great! Thanks, @DenisaCG!

I have a minor suggestion that I forgot to tell you before. Jupyterlab offers a set of UI components that we can reuse on certain occasions. In this case, we can easily reuse the run icon used in the notebook's toolbar.

We can import the run icon from `import { runIcon } from '@jupyterlab/ui-components';` and then use it directly in the button like this:

```suggestion
    const button = new ToolbarButton({
      label: 'Run Code',
      icon: runIcon,
      className: 'jp-blockly-button',
      onClick: runCode,
      tooltip: 'Run Code'
    });
    button.addClass('jp-blockly-runButton');
    this.toolbar.addItem('run', button);
```

Apart from the icon, there is something weird with the class ToolbarButton. One of its children is taking the on-hover action and modifying the color. We can do a workaround by adding the following CSS classes to the `base.css` file:

```
.jp-blockly-button:hover {
	background-color: transparent !important;
}

.jp-blockly-runButton {
	background-color: var(--md-green-500)
}

.jp-blockly-runButton:hover {
	background-color: var(--md-green-700)
}
```